### DISCUSSION TITLE
How to use localized Markdown files in Laravel

### DISCUSSION THREAD

**Discussion by trianity** [spatie/laravel-markdown#10] [Ideas]
For example, Grav CMS can handle the language version of .md files ( something.en.md , something.be.md , something.hu.md ) as a standard feature and the question is how to make Laravel-Markdown load the appropriate localized file into the blade template?

**Comment by freekmurze** [comment]
Right now this isn't possible.

I would accept a PR that adds an `input-file` attribute to the component. If a value is passed to that attribute, that file should be read and its contents should be used instead of the content of `$slot`
